GPU/GPUTracking/TRDTracking/GPUTRDInterfaces.h

```diff
@@ -139,35 +148,29 @@ namespace gpu
 {
 
 template <>
-class trackInterface<o2::dataformats::TrackTPCITS> : public o2::dataformats::TrackTPCITS
+class trackInterface<o2::track::TrackParCov> : public o2::track::TrackParCov
 {
  public:
-  trackInterface<o2::dataformats::TrackTPCITS>() = default;
-  trackInterface<o2::dataformats::TrackTPCITS>(const trackInterface<o2::dataformats::TrackTPCITS>& param) = default;
-  trackInterface<o2::dataformats::TrackTPCITS>(const o2::dataformats::TrackTPCITS& param) = delete;
-  trackInterface<o2::dataformats::TrackTPCITS>(const GPUTPCGMMergedTrack& trk)
+  trackInterface<o2::track::TrackParCov>() = default;
+  trackInterface<o2::track::TrackParCov>(const o2::track::TrackParCov& param) = delete;
+  trackInterface<o2::track::TrackParCov>(const o2::dataformats::TrackTPCITS& trkItsTpc) : o2::track::TrackParCov(trkItsTpc.getParamOut()), mTime(trkItsTpc.getTimeMUS().getTimeStamp()), mTimePrecise(true)
```

Comment:
The ITS-TPC track can have up to 1 \mus uncertainty, don't you account for it? You could probably use symmetric mTimeAdd/SubMax equal to `trkItsTpc.getTimeMUS().getTimeStamp()` multiplied by some NSigma settable parameter. Then the flag `preciseTimeKnown` would lose its meaning and you would need to treat all tracks on the same footing, looking just on the time uncertainty.

Reply:
I do not yet account for it, no. For the time uncertainty can I also query the `TrackTPCITS::timeEst` which is of type `TimeStampWithError`? Then I could set mTime[Add/Sub]Max from that error value and don't need another configurable NSigma parameter.
How large is the (maximum) uncertainty for CE-crossing TPC tracks? At the moment I would not shift these in z, even if the track time differs from the trigger time.

Reply:
@martenole : For the TPC only tracks: The TrackTPC has members `mDeltaTFwd` and `mDeltaTBwd` which indicates the maximum possible time to be shifted. This is also the case for CE-crossing tracks.

Reply:
Ok, I will check then how large this gets. Because at the moment I shift the tracks only in case they are A/C-side only (and from the side I determine the shift direction). If they are CE crossing I guess I will simply check the dip angle to determine the shift direction.

Reply:
Yes, timeEst gives both the time and error(sigma).
The error depends on pt, from a few 10ns to a few 100. You don't need to shift the track to the boundaries, it is enough to shift to the central value, but you can account it as an extra error in z.

Reply:
Both for ITS-TPC tracks and for CE-crossing TPC-only tracks I don't shift the tracks to the boundaries, but I increase the track uncertainty in Z instead.
Probably I need to rebase this PR again after https://github.com/AliceO2Group/AliceO2/pull/5994 is merged due to a conflict in `Detectors/TRD/workflow/CMakeLists.txt`